packages/cli/index.ts

```diff
@@ -115,14 +115,16 @@ const main = defineCommand({
 
           for (const p of paths) {
             const pJsonPath = path.resolve(p, "package.json");
-            const { name } = await importPackageJson(pJsonPath);
+            const { name } = await readPackageJSON(pJsonPath);
 
             if (compact) {
-              await verifyCompactMode(name);
+              if(name) {
+                await verifyCompactMode(name);
+              }
             }
 
             deps.set(
-              name,
+              name!,
```

Comment:
Should I have to do the same for this too? (check it name is defined)

Reply:
no just on line 119 put a condition and throw an error if it's not defined
so we use one condition instead of two

Reply:
Done